pactsdk/asset.py

```diff
@@ -153,3 +153,10 @@ def __eq__(self, other_asset: object) -> bool:
         if not isinstance(other_asset, Asset):
             return False
         return self.index == other_asset.index
+
+    def __hash__(self) -> int:
+        """Returns an int which is the asset_id of the :class:`Asset` current object
```

Comment:
Please follow the same convention we use throughout the project.
You can call `make docs` in the `docs` folder to examine if the documentation is properly built.